RecoBTag/TensorFlow/plugins/DeepFlavourTagInfoProducer.cc

```diff
@@ -209,20 +218,16 @@ void DeepFlavourTagInfoProducer::produce(edm::Event& iEvent, const edm::EventSet
               { return btagbtvdeep::sv_vertex_comparator(sva, svb, pv); });
     // fill features from secondary vertices
     for (const auto & sv : svs_sorted) {
-      if (reco::deltaR(sv, jet) > jet_radius_) continue;
+      if (Geom::deltaR(sv.position() - pv.position(), flip_ ? -jet_dir : jet_dir) > jet_radius_) continue;
```

Comment:
please stay with reco::deltaR.
Also, it makes more sense to use deltaR2 and save some CPU cycles